deploy/core-agents-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: core-agents
```

Comment:
agentS or agent?

Reply:
I used the plural because, although we only have a single container at the moment, as we create additional agents we could add additional containers to the deployment.